Documentation/troubleshooting.rst

```diff
@@ -249,6 +250,72 @@ following output in ``cilium status``:
     See how to :ref:`ensure a pod is managed by Cilium<ensure_managed_pod>`
     for more details.
 
+Observing flows with Hubble Relay
+=================================
+
+.. note::
+   **Hubble Relay** is beta software and as such is not yet considered
+   production ready.
+
+Hubble Relay is a service which allows to query multiple Hubble instances
+simultaneously and aggregate the results. As Hubble Relay relies on individual
+Hubble instances, Hubble needs to be enabled when deploying Cilium. In
+addition, the Hubble service needs to be exposed on TCP port ``4244``. This can
+be done via the Helm values ``global.hubble.enabled=true`` and
+``global.hubble.listenAddress=":4244"`` or the
+``--enable-hubble --hubble-listen-address :4244`` options on cilium-agent.
+
+.. note::
+   Enabling Hubble to listen on TCP port 4244 globally has security
+   implications as the service can be accessed without any restriction.
+
+Hubble Relay can be deployed using Helm by setting
+``global.hubble.relay.enabled=true``. This will deploy Hubble Relay with one
+replica by default. Once the Hubble Relay pod is running, you may access the
+service by port-forwarding it:
+
+.. code:: bash
+
+    $ kubect -n kube-system port-forward service/hubble-relay 4245:80
+
+This will forward the Hubble Relay service port (``80``) to your local machine
+on port ``4245``. The next step consists of downloading the latest binary
```

Comment:
I think this has the ports mixed up a bit. Maybe change to something as follows:
```suggestion
    $ kubect -n kube-system port-forward service/hubble-relay 80:4245

This will forward the Hubble Relay service port (``4245``) to your local machine
on port ``80``. The next step consists of downloading the latest binary
```

Might also be good to use an unprivileged port on the local machine instead of 80.

Reply:
From `kubectl port-forward -h`:
>  Listen on ports 5000 and 6000 locally, forwarding data to/from ports 5000 and 6000 in a pod selected by the service
  `kubectl port-forward service/myservice 5000 6000`

It's not messed up: the Hubble Relay service port is `80` and this command forwards it to the local port `4245`. I can pick a different port than `4245` as it's the Hubble Relay internal port if you think this is confusing but it somehow made sense in my head.

Reply:
Thanks for the explanation. Indeed I was confused by the Hubble Relay internal port. Not sure if it's worth to change to a different port given this is the troubleshooting section.